ledger/participant-state/kvutils/app/src/main/scala/com/daml/ledger/participant/state/kvutils/app/Config.scala

```diff
@@ -87,6 +88,7 @@ object Config {
       enableMutableContractStateCache = false,
       enableInMemoryFanOutForLedgerApi = false,
       enableHa = false,
+      maxDeduplicationDuration = None,
```

Comment:
As this option is related to ledger configuration, you may want to group it with `configurationLoadTimeout`.